addon/README.md

```diff
@@ -38,6 +39,8 @@ Emitted:
 * `addon-install:download-ended`
 * `addon-install:download-cancelled`
 * `addon-install:download-failed`
+* `addon-uninstalling`
+* `addon-uninstalled`
```

Comment:
`addon-uninstall:uninstall-started` / `addon-uninstall:uninstall-ended` would match the rest of the API, or you could change the old API to match this less verbose version. either way, aim to be consistent